### DISCUSSION TITLE
Nightscout Status has been taken down

### DISCUSSION THREAD

**Discussion by nielsmaerten** [nielsmaerten/nightscout-assistant#118] [General]
## Update (2021-10-19): Please switch to Gluco Check if you can
Nightscout Status won't be back, but you can use [Gluco Check](https://glucocheck.app) instead. Gluco Check is available in English and Dutch. For other languages, [please help us translate](https://pages.glucocheck.app/translations).

## Original post:
On Thursday October 14, I got an email from Google telling me they took down Nightscout Status.  

I'm obviously puzzled by this, because they didn't actually give any reason. The app was just taken down without any notice or explanation.
I will reach out to Google and ask for clarification, and hopefully we can get the app back up.  

I'll update this post when I hear back from them. 

_Niels_

Here's the full email I received:
![image](https://user-images.githubusercontent.com/4604406/137370833-286fe4cf-cea2-4a14-9937-4001eeabe172.png)

**Comment by Atlas-Night-Out** [comment]
who even has Done this needs to be saked it affect Diabetics lives and they need to be ashamed of themself!

**Reply by Atlas-Night-Out** [reply]
How do I delete Nightscout Status from Google home? I do not seem to have a remove option! Thanks

**Reply by nielsmaerten** [reply]
Since Google has taken down Nightscout Status, your account on Google Home has already been deleted. You don't have to do anything yourself.

If you also want to delete your nightscout information, you can log in on https://git.io/nightscoutstatus, remove your info and save the form with blank fields.

If you are using [Gluco Check](https://glucocheck.app) right now, then your entire account on Nightscout Status will be automatically deleted in the future as well.

**Comment by nielsmaerten** [comment]
# Update (2021-10-19)
## Google replied: Nightscout Status won't be back :(

Here's what they said: (emphasis mine)
> Dear Developer,
> Thank you for reaching out to Actions on Google Support Team!
> Please be advised that we checked your Action and _it does appear that your Action may have been taken down without cause_, can you please resubmit a new version of your Action so it can be reviewed once again.  
> If you have any additional questions, feel free to reach out to us.
> Regards,
> Actions on Google Support Team

"Taken down without cause"? Seriously?? 🤨🤨
And to get Nightscout Status back up, I have to resubmit it and go through the review process again.

### I simply don't have the time or resources to work on this right now.
This means, unfortunately, that Nightscout Status won't be back anytime soon.

# So, what should _you_ do?
## If you were using Nightscout Status in _English_ or _Dutch_:
Visit https://glucocheck.app.

Gluco Check was supposed to be the successor to Nightscout Status. It's also built by me. Just start using Gluco Check instead of Nightscout Status, and you'll have access to additional features. [Watch a demo here](https://youtu.be/o7-T2IrDJ_A).

## If you were using another language:
Unfortunately Gluco Check is not yet fully translated. If you want to help, visit: https://pages.glucocheck.app/translations.
The sooner we can get the app fully translated, the sooner we can make it available again in all languages.
